src/pages/Cart/index.js

```diff
@@ -11,6 +12,17 @@ function Checkout() {
 export default Checkout;
 
 function Cart() {
+  const store = useCartStore();
+  const cart = store.cart;
+
+  useEffect(function () {
+    document.title = "/checkout";
```

Comment:
Please, update the title. It should not have the `/`—the title is not the page route

<img width="884" alt="Screenshot 2024-09-24 at 22 06 40" src="https://github.com/user-attachments/assets/04351e1a-6dca-4250-bb37-37635277c154">

You can look at other websites to see what their titles are like